modules/cart/src/CartManager.php

```diff
@@ -135,6 +136,8 @@ public function updateLineItem(OrderInterface $cart, LineItemInterface $line_ite
     $line_item->save();
     $event = new CartLineItemUpdateEvent($cart, $line_item, $original_line_item);
     $this->eventDispatcher->dispatch(CartEvents::CART_LINE_ITEM_UPDATE, $event);
+    // We need to recalculate the order's total when updating line items.
+    $cart->save();
```

Comment:
Another thought, do we need to run save before passing to the event? Otherwise the `$cart` won't have an update order total.

I almost feel like we need a method on the order entity to refresh the order total without saving. @bojanz thoughts?